src/Spectre.libClassifier/Svm.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+* Svm.h
+* Simple wrapper for OpenCV SVM.
+*
+Copyright 2017 Spectre Team
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "Svm.h"
+#include "Spectre.libClassifier/OpenCvDataset.h"
+#include "Spectre.libClassifier/UnsupportedDatasetTypeException.h"
+
+namespace Spectre::libClassifier
+{
+Svm::Svm(uint iterationsLimit, double tolerance)
```

Comment:
`uint` should be replaced with `unsigned int`.

Reply:
done